src/scss/_variables.scss

```diff
@@ -68,3 +75,91 @@ $spacers: (
 $embed-responsive-aspect-ratios: ((16 9), (2 1));
 $enable-responsive-font-sizes: true;
 $rfs-factor: 4;
+
+//Global typography
```

Comment:
It does not look like these all these variables gets used either. i dont think we should have alot of variables in our system that is not used.